src/posit/connect/resources.py

```diff
@@ -234,3 +234,83 @@ def find_by(self, **conditions: Any) -> T | None:
         """
         collection = self.fetch(**conditions)
         return next((v for v in collection if v.items() >= conditions.items()), None)
+
+
+class _Resource(dict):
+    def __init__(self, ctx: Context, path: str, **attributes):
+        self._ctx = ctx
+        self._path = path
+        super().__init__(**attributes)
+
+    def destroy(self) -> None:
+        self._ctx.client.delete(self._path)
+
+    def update(self, **attributes):  # type: ignore
```

Comment:
Can we replace it with the pyright ignore command? `# pyright: ignore[RULE]`
(And if it's only parameter specific, can we put it on an isolated line only? Thank you!)